modules/custom/az_migration/README.md

```diff
@@ -46,20 +46,20 @@ CONFIGURATION
 
   ```
   $databases['migrate']['default'] = [
-    'driver' => 'my_driver',
-    'namespace' => 'Drupal\my_module\Driver\Database\my_driver',
-    'autoload' => 'modules/my_module/src/Driver/Database/my_driver/',
+    'driver' => 'mysql',
+    'namespace' => 'Drupal\core\Driver\Database\mysql',
     'database' => 'databasename',
-    'username' => 'sqlusername',
-    'password' => 'sqlpassword',
+    'username' => 'databaseusername',
+    'password' => 'databasepassword',
+    'port' => 'databaseport',
     'host' => 'localhost',
-    'prefix' => '',
+    'prefix' => 'tableprefix',
```

Comment:
Nitpick: table prefixes are a pretty seldom used feature (in my experience) so maybe it'd be best to just leave this one blank?
```suggestion
```